### PR TITLE
Card

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
https://www.loom.com/share/e00363ea72dd4770b811fc7449467d17

feature:added card components
             updated pageLayout 
              updated button css replaced some rems with pixel to make it working

### CODE REVIEW

File: Card/card.css [hunk 1]
```diff
@@ -0,0 +1,248 @@
+@import url(../pageLayout.css);
+@import url(../Button/button.css);
+:root {
+    --card-lower-background-color: #f5f5f5;
+    --card-lower-border-color: #474747;
+    --image-container-color: white;
+    --card-border-color: black;
+    --card-strike-color: red;
+    --card-success-color: green;
```
Comment (kotesh-arya) [inline Card/card.css:9]: It is better to give the HEXCODES for these gneric colours also.
CAUSE: when giving colour-value og BLUE, each browser has its own variant , then your colour will be shown differently on various browsers

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 10px;
+}
+
+.card.shopping_card.card_shadow.vertical .card-description .product_rating {
+    background-color: var(--card-success-color);
+    border-radius: 50%;
+    color: white;
```
Comment (kotesh-arya) [inline Card/card.css:125]: Be consistent with HEXCODES 👍🏻

File: Card/card.css [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border-radius: 10px;
+    background-color: gray;
+}
+
+.card .card-overlay-text span {
+    background-color: var(--primary-button-color);
+    opacity: 1 !important;
```
Comment (kotesh-arya) [inline Card/card.css]: I think It's said that , exclude !important for paroperties, try to avoid using it 👍🏻

**Review by kotesh-arya — CHANGES_REQUESTED** [review]

**Review by kotesh-arya — APPROVED** [review]
Good enough for a merge 👍🏻

**Review by anshtripathi01 — APPROVED** [review]
Use hex-code for color rest is good.

File: Card/card.css [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+    justify-content: center;
+    align-items: center;
+    padding: 4px 4px;
+    border: 4px solid var(--primary-button-color);
+}
+
+
+/* card with dismiss */
+
+.card.shopping_card.card_shadow.vertical .card_dismiss a {
```
Comment (AtulHande03a) [inline Card/card.css:136]: why to use so many classes for specificity ?? would directly using .card_dismiss a { properties} work???

**Pull request merged**